dbt_project.yml

```diff
@@ -12,4 +12,6 @@ data-paths: ["data"]
 models:
   recurly:
     vars:
-      subscriptions_table: # schema.table
+      subscriptions_table:          #table.schema
```

Comment:
this is stupid, but you have `table.schema` reversed. can you switch them so as not to confuse future users? :D